app/jobs/organization_export_job.rb

```diff
@@ -1,5 +1,5 @@
 class OrganizationExportJob < ApplicationJob
-  PART_SIZE = 1_000
+  PART_SIZE = ENV['BATCH_EXPORT_SIZE']
```

Comment:
Are you cool with moving this into the `secrets.yml` file? I think we sorta decided that using ENV outside that file was going to be an anti-pattern on this app.